website/docs/d/tenant_policies_ipsla_monitoring_policy.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "mso"
+page_title: "MSO: mso_tenant_policies_ipsla_monitoring_policy"
+sidebar_current: "docs-mso-data-source-tenant_policies_ipsla_monitoring_policy"
+description: |-
+Data source for IPSLA Monitoring Policy.
+---
+
+
+
+# mso_tenant_policies_ipsla_monitoring_policy #
+
+Data source for IPSLA Monitoring Policy.
+
+
+## Example Usage ##
+
+```hcl
+data "mso_tenant_policies_ipsla_monitoring_policy" "ipsla_policy" {
+  template_id        = mso_template.template_tenant.id
+  name               = "test_ipsla_policy"
+}
+```
+
+## Argument Reference ##
+
+* `template_id` - (Required) The unique ID of the template.
+* `name` - (Required) The name of the IPSLA monitoring policy.
+
+## Attribute Reference ##
+
+* `uuid` - (Read-Only) The UUID of the IPSLA monitoring policy.
```

Comment:
Is it intended to sometimes use IPSLA monitoring policy vs SLA vs Service vs nothing?